odkx-src/build-scripts.rst

```diff
@@ -67,6 +67,15 @@ If you are on :program:`Windows` use :file:`gradlew.bat` instead.
 
   If you are building with Android Studio, you will need to select the correct build variant. This is important when you don't have androidlibrary or androidcommon in your :ref:`build-scripts-directory-structure`. These are discussed more in the :ref:`next section <build-scripts-flavors>`.
 
+.. warning::
+
+  The Android tools have a release signature key unique to the ODK-X applications. Communication between the applications is secured by this key and can only happen when the application keys match. This ensures that the information passing between the applications cannot be intercepted by another application.
```

Comment:
This is not entirely accurate. The key only provides authentication for communication (IPC calls) between apps. It doesn't provide security for the data in-transit.

Reply:
~~signature key~~ signing key

It's called a signing key, which produces a signature.